bridges/prometheus/producer_test.go

```diff
@@ -181,15 +254,15 @@ func TestProduce(t *testing.T) {
 				},
 				Metrics: []metricdata.Metrics{
 					{
-						Name:        "test_histogram_metric",
+						Name:        "test_histogram_metric_with_exemplar",
 						Description: "A histogram metric for testing",
 						Data: metricdata.Histogram[float64]{
 							Temporality: metricdata.CumulativeTemporality,
 							DataPoints: []metricdata.HistogramDataPoint[float64]{
 								{
 									Count:        1,
 									Sum:          578.3,
-									Bounds:       []float64{0.005, 0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1, 2.5, 5, 10},
+									Bounds:       prometheus.DefBuckets,
```

Comment:
Doesn't `prometheus.DefBuckets` equal `[]float64{0.005, 0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1, 2.5, 5, 10}`? Changing this seems to obfuscate the expected value.

Reply:
reverted.